doc/how-to/getting_started_net_box.rst

```diff
@@ -0,0 +1,155 @@
+
+The next step is to create a new connection.
+In ``net.box``, self-connection is pre-established.
+That is, ``conn = net_box.connect('localhost:3301')`` can be replaced with the following command:
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn = net_box.self
+
+Then, make a ping:
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn:ping()
+    ---
+    - true
+    ...
+
+Using data operations
+---------------------
+
+The ``select()`` command below returns all tuples in the ``tester`` space where the key value is ``800``:
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn.space.tester:select{800}
+    ---
+    - - [800, 'TEST']
+    ...
+
+Insert two tuples into the space:
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn.space.tester:insert({700, 'TEST700'})
+    ---
+    - [700, 'TEST700']
+    ...
+    tarantool> conn.space.tester:insert({600, 'TEST600'})
+    ---
+    - [600, 'TEST600']
+    ...
+
+After the insert, we have one tuple where the key value is ``600``.
+To select this tuple, you can use the ``get()`` method.
+Unlike the ``select()`` command, ``get()`` returns only one tuple that satisfies the stated condition.
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn.space.tester:get({600})
+    ---
+    - [600, 'TEST600']
+    ...
+
+To update the existing tuple, you can use either ``update()`` or ``upsert()``.
+The ``update()`` method can be used for assignment, arithmetic (if the field is numeric),
+cutting and pasting fragments of a field, and deleting or inserting a field.
+
+The command below is used to update the tuple identified by primary key value = ``800``.
+The operation assigns a new value to the second field in the tuple:
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn.space.tester:update(800, {{'=', 2, 'TEST800'}})
+    ---
+    - [800, 'TEST800']
+    ...
+
+As for the ``upsert`` function, if there is an existing tuple which matches the key field of tuple, then the command
+has the same effect as ``update()``.
+Otherwise, the effect is equal to the ``insert()`` method.
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn.space.tester:upsert({500, 'TEST500'}, {{'=', 2, 'TEST'}})
+
+To delete a tuple, run the method below:
+
+..  code-block:: tarantoolsession
+
+    -- Delete tuples where the key value is 600
+    tarantool> conn.space.tester:delete{600}
+    ---
+    - [600, 'TEST600']
+    ...
+
+Then, replace the existing tuple with a new one
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn.space.tester:replace{500, 'New data', 'Extra data'}
+    ---
+    - [500, 'New data', 'Extra data']
+    ...
+
+Finally, select all tuples from the space:
+
+..  code-block:: tarantoolsession
+
+    tarantool> conn.space.tester:select{}
+    ---
+    - - [800, 'TEST800']
+      - [500, 'New data', 'Extra data']
+      - [700, 'TEST700']
+    ...
+
+In the end, close the connection when it is no longer needed:
```

Comment:
Let's separate this from data operations with a heading.

Reply:
Fixed